doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -137,16 +142,13 @@ Install ACRN on the Debian VM
       [    0.982837] ACRN HVLog: Failed to init last hvlog devs, errno -19
       [    0.983023] ACRN HVLog: Initialized hvlog module with 4 cp
 
-      $ uname -a
-      Linux debian 4.19.68-84.iot-lts2018-sos #1 SMP Debian 4.19.37-5+deb10u2 (2019-08-08) x86_64 GNU/Linux
-
-#. Enable the network sharing to give network access to User VM:
-
+Enable the network sharing to give network access to User VM
+************************************************************
    .. code-block:: bash
 
       $ sudo systemctl enable systemd-networkd
       $ sudo systemctl start systemd-networkd
 
-#. Prepare and Start a User VM.
-
-   .. important:: Need instructions for this.
+Prepare and Start a User VM
+***************************
+.. important:: Need instructions for this.
```

Comment:
We're still missing the instructions for preparing and starting a User VM.  Can you add it with this PR?

Reply:
@dbkinder 
**sudo make install** has copied acrn-dm and OVMF to the target directory.
And the correct IASL version was also build at previous step.
There is no more steps for  User VM setup.
Can we remove this item?
Thanks!

Reply:
I'm OK with removing this, or add a note to indicate the users can refer to our other documents on how to prepare a User VM image.

Reply:
> If there are indeed no more steps needed, then yes, you can update this PR and delete this heading and "important" note. @gvancuts do you agree?

Yes, please remove this @fuzhongl